source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample086.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dim"/>
+                     </specList></p>
+                     <p>An other way of encoding is to use the element <gi scheme="MEI">dim</gi>, that can be used to represent any single dimensional specification. The attribute <att>form</att> is required. When appropriate the elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, and <gi scheme="MEI">depth</gi> are preferred to specify the content of <gi scheme="MEI">dim</gi>.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample087.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="watermark"/>
+                     </specList></p>
+                     <p> Before starting to describe and encode watermarks, it is highly recommended to read up on filigranology separately. It is important to distinguish whether the watermark to be described consists of only one sign or is composed of main and countermarks. The positioning of the watermark on the manuscript page, the completeness as well as the periods of production and use also play a role.</p>
+                     <p>It is recommended to use at least the elements <gi scheme="MEI">title</gi>, <gi scheme="MEI">date</gi>, <gi scheme="MEI">annot</gi> and <gi scheme="MEI">locus</gi> in the description:</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample080.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p>For a detailed description or encoding of a watermark, respectively, <gi scheme="MEI">fig</gi> can be used, which on the one hand allows to refer to an existing image (<gi scheme="MEI">graphic</gi>) of the watermark and on the other hand increases the depth of the description. To mark up different components of a watermark, several <gi scheme="MEI">heraldry</gi> elements can be used in a <gi scheme="MEI">figDesc</gi> and related to each other by means of <gi scheme="MEI">relation</gi>.
```

Comment:
```suggestion
                     <p>For a detailed description or encoding of a watermark, <gi scheme="MEI">fig</gi> can be used, which on the one hand allows reference to an existing image (<gi scheme="MEI">graphic</gi>) of the watermark and on the other increases the depth of the description. To mark up different components of a watermark, several <gi scheme="MEI">heraldry</gi> elements can be used in a <gi scheme="MEI">figDesc</gi> and related to each other by means of <gi scheme="MEI">relation</gi>.
```

Reply:
```suggestion
                     <p>For a detailed description or encoding of a watermark, respectively, <gi scheme="MEI">fig</gi> can be used, which on the one hand allows to refer to an existing image (<gi scheme="MEI">graphic</gi>) of the watermark, and on the other hand increases the depth of the description. To mark up different components of a watermark, several <gi scheme="MEI">heraldry</gi> elements can be used in a <gi scheme="MEI">figDesc</gi> and related to each other by means of <gi scheme="MEI">relation</gi>.
```

Reply:
I can't get back to the first change to add this comma. When joining two long clauses with "and", it is helpful to add the comma. Although it is right to not overuse them.

And you would say "On the other hand" if starting a new sentence. Otherwise, "on the one hand . ..., on the other...." is sufficient.

Reply:
> Suggested change
>                      <p>For a detailed description or encoding of a watermark, respectively, <gi scheme="MEI">fig</gi> can be used, which on the one hand allows to refer to an existing image (<gi scheme="MEI">graphic</gi>) of the watermark and on the other hand increases the depth of the description. To mark up different components of a watermark, several <gi scheme="MEI">heraldry</gi> elements can be used in a <gi scheme="MEI">figDesc</gi> and related to each other by means of <gi scheme="MEI">relation</gi>.
>                      <p>For a detailed description or encoding of a watermark, <gi scheme="MEI">fig</gi> can be used, which on the one hand allows reference to an existing image (<gi scheme="MEI">graphic</gi>) of the watermark and on the other increases the depth of the description. To mark up different components of a watermark, several <gi scheme="MEI">heraldry</gi> elements can be used in a <gi scheme="MEI">figDesc</gi> and related to each other by means of <gi scheme="MEI">relation</gi>.

@mss2221 Is it really `on the other` without `hand` here?

Reply:
Parodon, I read only the first comment! It's fine, thanks for the explanation!